test/functional/apps/settings/_index_pattern_create_delete.js

```diff
@@ -0,0 +1,109 @@
+define(function (require) {
+  var Common = require('../../../support/pages/Common');
+  var SettingsPage = require('../../../support/pages/SettingsPage');
+  var expect = require('intern/dojo/node!expect.js');
+  var Promise = require('bluebird');
+
+  return function (bdd) {
+    bdd.describe('creating and deleting default index', function describeIndexTests() {
+      var common;
+      var settingsPage;
+      var remote;
+
+      bdd.before(function () {
+        common = new Common(this.remote);
+        settingsPage = new SettingsPage(this.remote);
+        remote = this.remote;
+      });
+
+
+      bdd.describe('index pattern creation', function indexPatternCreation() {
+        bdd.beforeEach(function be() {
```

Comment:
Some of the tests do modify the kibana index. For example, when index patterns are created and/or removed. I asked @LeeDr about this previously as well, and his intention is that every test starts in a clean, fresh state, which makes sense. 

You're right, most tests just need Kibana to be online, so the index simply needs to exist in whatever state. We could make that a suite based operation, instead of always doing it every time.